ThingIFSDK/ThingIFSDK/QueryClause.swift

```diff
@@ -0,0 +1,368 @@
+//
+//  QueryClause.swift
+//  ThingIFSDK
+//
+//  Created on 2017/01/23.
+//  Copyright (c) 2017 Kii. All rights reserved.
+//
+
+import Foundation
+
+/** Base class for query clause classes.
+
+ This is abstract class. Developers does not need to directly use this
+ class.
+ */
+open class QueryClause: NSObject, BaseClause {
```

Comment:
I see. It is okay then. You already made the call of methods of these classes failure.